src/StableMemory.mo

```diff
@@ -0,0 +1,93 @@
+/// Byte-level access to (virtual) _stable memory_.
+///
+/// This is a lightweight abstraction over IC _stable memory_ and supports persisting
+/// raw binary data across Motoko upgrades.
+/// Use of this module is fully compatible with Motoko's use of
+/// _stable variables_, whose persistence mechanism also uses (real) IC stable memory internally, but does not interfere with this API.
```

Comment:
Is the stable variables mechanism extended to the big-mem already?

Reply:
No, its not, but I think we are ok, unless you've got something specific in mind. I think the intention is to add a StableMemory64 with 64 bit offsets when we get to that point.

I guess this means these operations (like the ic0.stable_read/write) will start failing once we allocate more that 4GB of (real) stable memory, in which case the user should be using the StableMemory64 library...